app/src/main/java/com/arctouch/codechallenge/Details/DetailsMoviePresenter.kt

```diff
@@ -0,0 +1,31 @@
+package com.arctouch.codechallenge.Details
+
+import com.arctouch.codechallenge.api.API_KEY
+import com.arctouch.codechallenge.api.Client
+import com.arctouch.codechallenge.api.DEFAULT_LANGUAGE
```

Comment:
I usually use an android studio shortcut that removes imports that are not being used, I find it bad to leave this type of import unused because it increases the amount of lines in the file.


In that case, I've been optimizing the code, and I've forgotten to remove those imports.